core/pom.xml

```diff
@@ -192,7 +192,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.17</version>
```

Comment:
It looks like surefire plugin upgrade and adding junit-jupiter-api makes all tests run through junit5 and that affects logging (there's way less messages in test output, no trace of >>>> & <<<<) and run time (it takes ages for all tests to finish on windows). 

Funny side effect : this PR bumps the number of tests by ~ 200k .

Reply:
Yes, the logging is affected but I would welcome solutions here. Our specific stdout is consumed by junit5. System out print gets output no problem. At the same time error logging is let thru. Mystery.

Reply:
Test on windows seem to run same as before.

https://dev.azure.com/questdb/questdb/_build/results?buildId=10256&view=results

Reply:
i shall investigate the logging situation, worst case scenario - rewrite tests in Junit4

Reply:
done in https://github.com/questdb/questdb/pull/2527/commits/c84fbc9604fa6791384002d3b9a1b38b41759d9d